docs/UserGuide.md

```diff
@@ -49,41 +49,64 @@ See [How do I move a file?](#q-how-do-i-move-a-file) for help.
-                    dialog box that opens and press "Enter" on your keyboard to open up a terminal window.</li>
+
+        <li><strong>For Mac users:</strong> open the Terminal app on your Mac</li>
+        <li><strong>For Window users:</strong> press the windows key, type `powershell` and press "Enter" on your keyboard</li>
+        <li><strong>See <a href="#faq">How do I open a terminal?</a> for more information.</li>
     </ul>
-    See <a href="#faq">How do I open a terminal?</a> for more information.
 </box>
 
 3. Use `cd` command in the terminal to navigate to the same location as `hrmate.jar` in step 1. See [How do I navigate files in terminal?](#faq)
 
-4. Ensure you have Java `11` or above installed in your Computer. This can be done by typing `java -version` in your terminal and pressing "Enter".
- - See <a href="#faq">How do I download Java 11?</a> if Java is not updated or if the terminal does not recognise the `java` input. 
+4. Ensure you have `Java 11` or above installed on your Computer. This can be done by typing `java -version` in your terminal.
+ - See <a href="#faq">How do I download Java 11?</a> if Java is not updated or if the terminal does not recognize the `java` input. 
 
 5. Type `java -jar hrmate.jar` into the terminal and press "Enter" on your keyboard.<br>
 An app similar to the one below should appear in a few seconds. The app is populated with some sample data for you to experiment with.<br>
 We have added some annotations in <span style="color:red">red</span> so that you can understand the app visually.
 ![Ui](images/Ui-annotated.png)
 
-6. Try typing some commands into the command box and pressing "enter" to execute. Here are some commands to start with.
- - `help` : Opens the help window.
- - `list` : Lists all contacts.
- - `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : Adds a contact named `John Doe` to the employee list
- - `add-leave 1 title/medical leave start/2023-11-11 end/2023-11-11` : Adds an leave entry for the first employee in the current employee list. Note that the default leave status is `PENDING`.
- - `delete 3` : Deletes the 3rd employee shown in the current employee list.
- - `clear` : Deletes all employees and leaves.
- - `exit` : Exits the app.
+6. You can try typing some commands into the command box and pressing "enter" to execute. We have listed down some commands for you to try. <br>
+ - `help`: Opens the help window.
+ - `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`: Adds an employee named `John Doe` to the employee list.
+ - `list`: Lists all employees.
+ - `add-leave 1 title/medical leave start/2023-11-11 end/2023-11-11`: Adds a leave entry for the first employee in the current employee list. Note that the default leave status is `PENDING`.
+ - `delete 2`: Deletes the second employee shown in the current employee list.
```

Comment:
Maybe it is better if the description here is `deletes the second employee record, including second employee shown in the current employee list and leaves related to the employee in the leave list?